addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2739,6 +2739,19 @@ var SnippetsMenu = Widget.extend({
             forceEmptyTab: true,
         });
     },
+    /**
+     * Activate the Blocks tab.
+     *
+     * @private
+     */
+    _activateBlocksTab() {
+        this._activateSnippet(false).then(() => {
```

Comment:
Even without my first comment, best to `return` the promises created by a method in case someone wants to use it and actually awaits the inner operations.